package.json

```diff
@@ -65,10 +65,8 @@
   },
   "devDependencies": {
     "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/stripes": "^1.0.0",
```

Comment:
We also need it listed in peerDependencies for use within a platform

Reply:
👍 yes we do